test/architecture-tests/src/test/java/software/amazon/awssdk/archtests/CodingConventionWithSuppressionTest.java

```diff
@@ -52,7 +53,9 @@ public class CodingConventionWithSuppressionTest {
                       ArchUtils.classNameToPattern("software.amazon.awssdk.services.s3.internal.crt.S3CrtResponseHandlerAdapter")));
 
     private static final Set<Pattern> ALLOWED_ERROR_LOG_SUPPRESSION = new HashSet<>(
-        Arrays.asList(ArchUtils.classNameToPattern(EmfMetricLoggingPublisher.class)));
+        Arrays.asList(
+            ArchUtils.classNameToPattern(EmfMetricLoggingPublisher.class),
+            ArchUtils.classWithInnerClassesToPattern(ResponseTransformer.class)));
```

Comment:
This was existing usage of Logger.error that was not caught by this because it was in a lambda.